recipes/cjson/all/conanfile.py

```diff
@@ -78,17 +77,11 @@ def package(self):
         cmake.install()
         tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
-        self._move_dll_to_bin_folder()
-
-    def _move_dll_to_bin_folder(self):
-        if self.settings.os == "Windows" and self.options.shared:
-            bin_dir = os.path.join(self.package_folder, "bin")
-            if not os.path.exists(bin_dir):
-                os.mkdir(bin_dir)
-            for dll_file in glob.glob(os.path.join(self.package_folder, "lib", "*.dll")):
-                shutil.move(dll_file, bin_dir)
 
     def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "cjson"
+        self.cpp_info.names["cmake_find_package_multi"] = "cjson"
```

Comment:
`cjson` is already the name of the package. Am I missing something?

```suggestion
```

Reply:
No, these extra names are not required indeed.

Reply:
I know, but when I read a conanfile without `cmake_find_package`, `cmake_find_package_multi` or `pkg_config`, I always wonder "is it because exported name is the name of the recipe or because it's missing?".
By enforcing those names when upstream explicitly defines them (even if default is fine), it avoid confusion for recipe maintainers.

Reply:
I know that feeling... from the consumer POV it is the reason why I wrote the `markdown` generator 😄